### PR TITLE
Improve the Core installation and resolve one troubleshooting

### PR DESCRIPTION
**Pull request by jazzi**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @jazzi, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22371).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_includes/installation/core.md [hunk 1]
```diff
@@ -102,3 +113,37 @@ If this address doesn't work you may also try `http://localhost:8123` or `http:/
 When you run the `hass` command for the first time, it will download, install and cache the necessary libraries/dependencies. This procedure may take anywhere between 5 to 10 minutes. During that time, you may get "site cannot be reached" error when accessing the web interface. This will only happen for the first time, and subsequent restarts will be much faster.
 
 </div>
+  
+### Starting Home Assistant on boot
```
Comment (frenck) [inline source/_includes/installation/core.md:117]: We consider this out of scope for our documentation. This is an advanced installation method that covers setting up Home Assistant itself, not how to administer your system.

These section have been removed from our documentation as is available on the community guides section on the forums (as an editable wiki).

File: source/_includes/installation/core.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ sudo apt-get upgrade -y
 Install the dependencies:
 
 ```bash
-sudo apt-get install -y python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg0-dev tzdata
+sudo apt-get install -y python3 python3-dev python3-venv python3-pip libffi-dev libssl-dev libjpeg-dev zlib1g-dev autoconf build-essential libopenjp2-7 libtiff5 libturbojpeg0-dev tzdata cargo ndg-httpsclient pyopenssl pyasn1
```
Comment (frenck) [inline source/_includes/installation/core.md:36]: Why are these added? This is system-dependent?

This is not needed on the latest Debian (which is the reference) if I'm correct?

File: source/_includes/installation/core.md [hunk 3]
```diff
@@ -87,6 +87,17 @@ Once you have installed the required Python package, it is now time to install H
...
+  
+If you encounter error *No space left on device*, probably there is limitation of /tmp, try moving this directory to elsewhere.
+  
+```text
+  
+mkdir -p /home/homeassistant/tmp
+export TMPDIR=/home/homeassistant/tmp
+  
+```  
+</div>
```
Comment (frenck) [inline source/_includes/installation/core.md:100]: This is not related to Home Assistant, but to system administration in general, which is not a responsibility for our documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_includes/installation/core.md (hunk 2 above)
Comment (jazzi) [reply source/_includes/installation/core.md:36]: I am on Armbian latest which is based on Debian bullseye, it complains lack of Rust Complier even install it manually but works fine if cargo installed.

ndg-httpsclient pyopenssl and pyasn1 are used to resolve a SSL problem I encountered that can not download from GitHub.com.

**Comment by jazzi** [comment]
As @frenck declared this document is mostly about setting up not how to administer, thus no need for this pull request, thanks for your time.

**Comment by frenck** [comment]
Nevertheless, thanks for willing to contribute @jazzi 👍 
Always appreciated ❤️ 

../Frenck
